pom.xml

```diff
@@ -396,6 +396,30 @@
             <autoVersionSubmodules>true</autoVersionSubmodules>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.eclipse.m2e</groupId>
+          <artifactId>lifecycle-mapping</artifactId>
```

Comment:
Thanks for including the link in the commit message. As I understand it, this doesn't actually change how the build proceeds, but only adds some metadata for when Eclipse opens the project.